CONTRIBUTING.md

```diff
@@ -189,10 +189,28 @@ executable and the VSCode plugin.
   
 ### VSCode Plugin
 
+#### Prerequisites
+
+- [vsce](https://github.com/microsoft/vscode-vsce)
+- [yalc](https://github.com/wclr/yalc)
```

Comment:
Just a note: `yalc` is a bit of a weird dependency here because we only need to run `yalc remove` if `yalc add` was run before. If someone doesn't even have it installed, then they shouldn't worry about it. But, since we are using `yalc` on the script, they'd have to install it in order to run the script.

I don't think that's necessary for now, but at some point we could try to remove this requirement by changing the script to something like "if there are traces of yalc usage (i.e. files), run `yalc remove`"

Reply:
Thanks for the info!

This is indeed currently recording what is needed to run the script (I hope :) ).